.github/workflows/eval-prod.yml

```diff
@@ -0,0 +1,33 @@
+name: Run Eval Benchmark Prod
+
+env:
+  EVAL_API_URL: ${{ secrets.EVAL_API_URL }}
+  EVAL_API_KEY: ${{ secrets.EVAL_API_KEY }}
+  EVAL_EXPERIMENT_ID: ${{ secrets.EVAL_BENCHMARK_EXPERIMENT_ID }}
+
+on:
+  workflow_run:
+    workflows: ["Deploy Images to GHCR"]
+    types:
+      - completed
+    branches:
+      - main
+
+
+jobs:
+  run-eval-benchmark-prod:
+        runs-on: ubuntu-latest
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        steps:
+          - name: 'Install dependencies'
+            run: |
+              python -m pip install --upgrade pip
+              pip install requests
+
+          # make sure the image is deployed before running the eval benchmark
+          - name: Wait for 2 minutes
+            run: sleep 120
```

Comment:
This depends on @tomkosm fixing Keel on `firecrawl-worker` that's been broken for months :D